draft-thomson-use-it-or-lose-it.md

```diff
@@ -185,7 +202,12 @@ element on path can interact with the protocol.  For example, HTTP was
 specifically designed with intermediation in mind, transparent proxies
 {{?HTTP=RFC7230}} are not only possible but sometimes advantageous, despite
 some significant downsides.  Consequently, transparent proxies for cleartext
-HTTP are commonplace.
+HTTP are commonplace.  The DNS protocol was designed with
+intermediation in mind through its use of caching recursive
+resolvers {{?DNS=RFC1034}}.  What was less anticipated was the forced
+spoofing of DNS records by many middle-boxes to inject authentication
+or pay-wall mechanisms as an authentication and authorization check,
+which are now prevalent in hotels, coffee shops and business networks.
```

Comment:
This doesn't really cover all the reasons.  Spoofing can be "helpful", or it can be used to enforce policy as well.  Maybe the right thing to do here is to stop after "many middleboxes".

Reply:
I was trying to give an example that the reader would understand and latch on to as an example.  One of the issues I'm trying to address in many of my changes is to introduce specific examples, rather than references to protocols that they may not know the details of.  But your point is valid that it's not the only case.  How about s/to inject/such as those that inject/

Reply:
SGTM

Reply:
changed